src/components/CoordinatorHome.js

```diff
@@ -4,149 +4,209 @@ import styled from "styled-components"
 
 import ReactTable from "react-table"
 import "react-table/react-table.css"
+import { blue, darkblue } from "../colors"
+
+import { Input } from "reakit";
+import { Block, Button, Backdrop, Portal, Overlay } from "reakit";
+
+// import Button from "../primitives/Button"
+
+// src/primitives/Button.js
+
+
+// For displayig image
+import { Image } from "reakit"
+import strip_report from "../images/strip_report.jpg"
 
 import { green, red, white, darkgrey } from "../colors"
 
 import Heading from "../primitives/Heading"
+import Help from    "../primitives/Help"
+import PhotoPopout from "../primitives/PhotoPopout"
+
+// make my own "Photo" popup?
+// import Photo from   "../pr"
 
 import { Icon } from "@mdi/react"
 import { mdiClose, mdiCheckCircle, mdiCheckCircleOutline } from "@mdi/js"
 
 import moment from "moment"
 
+// TODO: do we need these?
 const Icons = {
   good: <Icon size={1} color={green} path={mdiCheckCircle} />,
   okay: <Icon size={1} color={green} path={mdiCheckCircleOutline} />,
   bad:  <Icon size={1} color={red}   path={mdiClose} />,
 }
 
-const CoordinatorHome = observer(({ store }) => (
-  <Layout>
-    <span>{moment().format("YYYY-MM-DD")}</span>
-    <Heading>Manage Patient Progress</Heading>
+const CoordinatorHome = observer(({ store }) => {
 
-    <Table
-      data={store.coordinator.patients}
-      columns={[
+  const tableColumns= [
+    {
+      Header: "Participant Info",
+      columns: [
         {
-          Header: "medication_report_dates",
+          Header: "Status",
           accessor: "medication_report_dates",
-
-          Cell: patient => {
-            // TODO placeholder,
-            // until we get the data hooked up for this.
-            let random = Math.random()
-            let randomHealth =
-              random > 0.3 ?  ( random > 0.6 ?  "good" : "okay") : "bad"
-            let healthBadge = Icons[randomHealth]
-
-            return (
-              healthBadge
-            )
-          }
+          width: 110,
+          // Link this up to the patient so that it changes after a test result is submitted
+          Cell: row => (
+            <span>
+              <span style={{
+                color: row.value === 'notreported' ? '#ff2e00'
+                  : row.value === 'reported' ? '#ffbf00'
+                  : '#57d500',
+                transition: 'all .3s ease'
+              }}>
+                &#x25cf;
+              </span> {
+                row.value === 'notreported' ? 'Not Reported'
+                : row.value === 'reported' ? `Reported`
+                : 'Confirmed'
+              }
+            </span>
```

Comment:
You may need to wrap this `Cell` component in an `observable()` function.

It would be nice if we could separate the JSX here from the Javascript data structure.

Would it be possible to move this component into another file, like `src/components/PatientInfo.js`?